apps/institution-service/src/controllers/institutionController.test.ts

```diff
@@ -87,7 +87,9 @@ describe("institutionController", () => {
       // eslint-disable-next-line @typescript-eslint/unbound-method
       expect(res.status).toHaveBeenCalledWith(201);
       expect(res.json).toHaveBeenCalledWith(
-        expect.objectContaining(institutionBody),
+        expect.objectContaining({
+          institution: expect.objectContaining(institutionBody),
+        }),
```

Comment:
Just a question because of my ignorance. Is this a common pattern?--nested "expects"? Very cool. I didn't realize you could do that.

Reply:
I'm not sure how common it is but I've found it to be pretty useful